packages/web-components/fast-foundation/src/combobox/combobox.template.ts

```diff
@@ -1,48 +1,47 @@
 import { html, ref, slotted } from "@microsoft/fast-element";
 import type { ViewTemplate } from "@microsoft/fast-element";
+import type { FoundationElementTemplate } from "../foundation-element";
 import { Listbox } from "../listbox/listbox";
 import { endSlotTemplate, startSlotTemplate } from "../patterns/start-end";
-import type { ElementDefinitionContext } from "../design-system";
 import type { Combobox, ComboboxOptions } from "./combobox";
 
 /**
  * The template for the {@link @microsoft/fast-foundation#(Combobox:class)} component.
  * @public
  */
-export const comboboxTemplate: (
-    context: ElementDefinitionContext,
-    definition: ComboboxOptions
-) => ViewTemplate<Combobox> = (
-    context: ElementDefinitionContext,
-    definition: ComboboxOptions
-) => html`
+export const comboboxTemplate: FoundationElementTemplate<
+    ViewTemplate<Combobox>,
+    ComboboxOptions
+> = (context, definition) => html`
     <template
+        aria-disabled="${x => x.ariaDisabled}"
```

Comment:
does having this at the root create issues with us delegating focus to the input below?

Reply:
The `disabled` property works on a [small list of elements and form-associated custom elements](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#enabling-and-disabling-form-controls:-the-disabled-attribute:the-button-element). Some browsers don't support `ElementInternals`, so we can't guarantee that the `disabled` attribute on its own suffices in every scenario.

Regardless, the `disabled` property does work on the shadow `<input>`. And while the `tabindex` prevents the entire component from being focusable while disabled, Narrator does at least announce it when reading the page in scan mode.